Apps/frontend/src/views/ControlPanelBottom.svelte

```diff
@@ -9,15 +9,22 @@
     MIN_CONTROL_PANEL_BOTTOM_HEIGHT,
     MIN_SWEEP_SLIDER_VALUE,
     NUM_CHANNELS,
+    TIME_PER_DIV,
   } from "../const";
   import {
     amplitudeAdjustment,
     expandedPanelOpen,
     offsetAdjustment,
     timeSweep,
   } from "../stores";
+  import { computeDisplayDeltaFromTimeSweep } from "../helper";
 
   export let controlPanelBottomHeight = 0;
+
+  const computeDisplaySpeed = (value) => {
+    let delta = computeDisplayDeltaFromTimeSweep(value);
```

Comment:
Not a big fan of splitting this into two similar named functions, but fine if it gets the jobs done.